Source/Core/DiscIO/NANDImporter.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <functional>
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+
+namespace DiscIO
+{
+class NANDImporter final
+{
+public:
```

Comment:
The constructor and destructor should be defaulted in the cpp file, since the class body contains non-trivial members.